analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -414,6 +414,122 @@ public void Invocation_IsNullOrEmpty_TryFinally()
             });
         }
 
+#if NET
+
+        [TestMethod]
+        public void Invocation_TryParse_Null()
```

Comment:
These are not about `TryParse` functionality, but about `NotNullWhen`
```suggestion
        public void Invocation_NotNullWhen_Null()
```